tardis/plasma/tests/test_hdf_plasma.py

```diff
@@ -1,19 +1,14 @@
-import os
 import pandas as pd
 import pytest
 
-from numpy.testing import assert_almost_equal
-import pandas.testing as pdt
-from tardis.plasma.properties import property_collections
-
 ###
 # saving and loading of plasma properties in the HDF file
 ###
 
 
-@pytest.fixture(scope="module", autouse=True)
-def to_hdf_buffer(hdf_file_path, simulation_verysimple):
-    simulation_verysimple.plasma.to_hdf(hdf_file_path, overwrite=True)
+# @pytest.fixture(scope="module", autouse=True)
+# def to_hdf_buffer(hdf_file_path, simulation_verysimple):
+#     simulation_verysimple.plasma.to_hdf(hdf_file_path, overwrite=True)
```

Comment:
This should just be removed

Reply:
Done in https://github.com/tardis-sn/tardis/pull/2413/commits/50365c2d325f9c56f1eab8cb879fe0c2bebaeaa6